### PR TITLE
CASMINST-4709 export token for ntp script

### PR DESCRIPTION
**Pull request by jacobsalmela**
Signed-off-by: Jacob Salmela <jacob.salmela@hpe.com>

## Summary and Scope

Adds an explicit step to export the token for use in `csm_ntp.py`

## Issues and Related PRs

* Resolves CASMINST-4709

### Tested on:

  * `#redbull`

### Test description:

```
ncn-s004:~ # TOKEN=token
ncn-s004:~ # /srv/cray/scripts/common/chrony/csm_ntp.py
You must set BSS TOKEN environment variable
ncn-s004:~ # export TOKEN
ncn-s004:~ # /srv/cray/scripts/common/chrony/csm_ntp.py
Chrony configuration created
Problematic config found: /etc/chrony.d/cray.conf.dist
Problematic config found: /etc/chrony.d/pool.conf
Restarted chronyd
```

## Risks and Mitigations

Low.

## Pull Request Checklist

- [ ] Version number(s) incremented, if applicable
- [ ] Copyrights updated
- [ ] License file intact
- [ ] Target branch correct
- [ ] CHANGELOG.md updated
- [ ] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: operations/node_management/Configure_NTP_on_NCNs.md [hunk 1]
```diff
@@ -77,6 +77,12 @@ The following steps are structured to be executed on one node at a time. However
 
 1. Set a token as described in [Identify Nodes and Update Metadata](Rebuild_NCNs/Identify_Nodes_and_Update_Metadata.md).
```
Comment (mharding-hpe) [inline operations/node_management/Configure_NTP_on_NCNs.md:78]: Should this step and the next one be done on each affected NCN? If so, you should put the "On each affected NCN" part above all of these steps.

File: operations/node_management/Configure_NTP_on_NCNs.md [hunk 2]
```diff
@@ -77,6 +77,12 @@ The following steps are structured to be executed on one node at a time. However
...
 
+1. Export the token.
+
+    ```bash
+    export TOKEN
```
Comment (mharding-hpe) [inline operations/node_management/Configure_NTP_on_NCNs.md:83]:
```suggestion
    ncn# export TOKEN
```

**Review by mharding-hpe — CHANGES_REQUESTED** [review]

File: operations/node_management/Configure_NTP_on_NCNs.md (hunk 1 above)
Comment (jacobsalmela) [reply operations/node_management/Configure_NTP_on_NCNs.md:78]: There's two contexts.  The first is running it against every node at once in the `for` loop.  The other is ad-hoc on the node you want.  In that instance, it mentioned the "affected ncn" https://github.com/Cray-HPE/docs-csm/pull/1705/files/8c69cbd5cac23171773edda446ac908539c7aa5f#diff-bdc6d5e32445a6f32b36d568145774ebe9b424e01dded6bb3c1713eabbc676a8R86

Comment (mharding-hpe) [reply operations/node_management/Configure_NTP_on_NCNs.md:78]: Okay, but as written the doc says to set the token and then, on all of the affected nodes, run that script. Do you not need to set the token on all of those nodes?

**Review by mharding-hpe — CHANGES_REQUESTED** [review]

File: operations/node_management/Configure_NTP_on_NCNs.md (hunk 1 above)
Comment (jacobsalmela) [reply operations/node_management/Configure_NTP_on_NCNs.md:80]: I see what you mean I think.  Check out the latest push

Comment (mharding-hpe) [reply operations/node_management/Configure_NTP_on_NCNs.md:80]: Yup, that's what I was trying to suggest originally, but I can see how my wording was not ideal.

**Review by johren-hpe — APPROVED** [review]

**Review by rustydb — APPROVED** [review]

File: operations/node_management/Configure_NTP_on_NCNs.md [hunk 3]
```diff
@@ -75,9 +75,17 @@ The following steps are structured to be executed on one node at a time. However
...
 
+On each affected NCN run the following:
+
 1. Set a token as described in [Identify Nodes and Update Metadata](Rebuild_NCNs/Identify_Nodes_and_Update_Metadata.md).
 
-1. On each affected NCN, run:
+1. Export the token.
+
+    ```bash
+    ncn# export TOKEN
```
Comment (jasonrouault) [inline operations/node_management/Configure_NTP_on_NCNs.md:85]: should we show a sample value? or that a value is expected?  e.g.  {enter-your-token-value-here}

**Pull request merged**